src/client/app/components/MeterAndGroupSelectComponent.tsx

```diff
@@ -23,37 +26,88 @@ import { labelStyle } from '../styles/modalStyle';
 
```

Comment:
Adding comment here since unsure where it should go: When you select a meter or group then it is displayed at the top of the box with an x. The item is labeled but the superscript m or g is gone. Is it possible to have it show?